pkg/p2p/testbed/kadcast_test.go

```diff
@@ -139,6 +139,9 @@ func assertBroadcastMsgReceived(t *testing.T, cluster []*testNode, sender int, d
 	blob, _ := crypto.RandEntropy(uint32(msgSize))
 	cluster[sender].Broadcast(context.Background(), blob)
 
+	if is_remote {
+		blob = blob[:1000]
```

Comment:
Any particular reason why message truncation is chosen here instead of  a simple 32 hash of the payload? For integrity, I'd use hash.

Reply:
Moved to blake2b 256hash